app/Http/Controllers/Backend/Social/MastodonProfileDetails.php

```diff
@@ -23,7 +23,7 @@ public function getProfileUrl(): ?string {
     }
 
     private function getData(): ?array {
-        return Cache::remember('mastodon_'.$this->user->username, 6000, function () {
+        return Cache::remember("mastodon_{$this->user->username}", 60 * 60 /* 1 hour */, function() {
```

Comment:
NTH: i would prefer sprintf over double quote strings with variables but this also works.